api-module-library/sharepoint/README.md

```diff
@@ -0,0 +1,18 @@
+# salesforce
+
+This is the API Module for salesforce that allows the [Frigg](https://friggframework.org) code to talk to the salesforce API.
+
+Read more on the [Frigg documentation site](https://docs.friggframework.org/api-modules/list/salesforce
```

Comment:
You can set the link to microsoft-sharepoint (even though the page doesn't yet exist)